grafana/dashboards/applications/rw-dashboard.json

```
@@ -200,7 +200,7 @@
           },
           "editorMode": "code",
           "exemplar": false,
-          "expr": "sort(max_over_time(timestamp(sum by(client) (changes(openhim_transactions_total{client!=\"undefined\"}[5m])) > 0)[$__range:1m]) * 1000)",
+          "expr": "(time() * 1000 - sort(max_over_time(timestamp(sum by(client) (changes(openhim_transactions_total{client!=\"undefined\"}[5m])) > 0)[$__range:1m]) * 1000)) / (1000*60*60*24)",
```

It looks like you might be able to `* 1000` since it appears on both sides of the equation which might make it simpler. I.e.

```suggestion
          "expr": "(time() - sort(max_over_time(timestamp(sum by(client) (changes(openhim_transactions_total{client!=\"undefined\"}[5m])) > 0)[$__range:1m]) )) / (60*60*24)",
```

Note: I haven't tested this as I don't have data so please check it.

---

Thanks @rcrichton , that's a nice improvement. I pushed the change; and also did test with data, all good